### DISCUSSION TITLE
AI會畫圖!!!

### DISCUSSION THREAD

**Discussion by harrylee1971** [1101-GS4719-C/Python-Community-GS4719-C#937] [General]
https://technews.tw/2022/12/25/can-ai-replace-human-jobs/
好猛好猛，現在AI的進步已經到可以繪畫出一張比我畫的好的圖出來了xd
不知道以後會進步到甚麼程度
但電影中機器人AI的三大原則到現在來看還是十分合理欸
希望能在安全的環境下一直進步

**Comment by StevenWanglolz** [comment]
我最近看了一部影片太討論藝術家該如何面對AI製圖的帶來的衝擊，我在留言區看到了這一段話"I'm motivated by spite honestly. I refuse to let some soulless algorithm outdo me as a flesh and blood being with a mind and soul. I believe human artists are always superior because of that, and I'm not willing to give up. AI will never be the next Beksinski or Goya, but people like us can be"如果你是藝術家或學習繪畫的人，千萬不要因為這科技就氣餒啊!!!
影片連結: https://www.youtube.com/watch?v=KU5rSzUXP9E

**Comment by CPCpyt** [comment]
我也覺得這是重要的科技趨勢，相信能夠帶給創作者或是普通人非常重要的啟發。
